tests/dip_sip/test_dip_sip.py

```diff
@@ -0,0 +1,150 @@
+import os
+import pytest
+import ptf.testutils as testutils
+import ptf.packet as packet
+from ipaddress import ip_address
+
+TOPO_LIST = {'t0', 't1', 't1-lag'}
+PORTS_TOPO = {'t1'}
+LAG_TOPO = {'t0', 't1-lag'}
+DEFAULT_HLIM_TTL = 64
+WAIT_EXPECTED_PACKET_TIMEOUT = 5
+
+
+@pytest.fixture(scope='function', autouse=True)
+def prepare_ptf(testbed_devices):
+    ptfhost = testbed_devices["ptf"]
+    # remove existing IPs from ptf host
+    ptfhost.script('scripts/remove_ip.sh')
+    # set unique MACs to ptf interfaces
+    ptfhost.script('scripts/change_mac.sh')
+
+
+def lag_facts(dut):
+    facts = {}
+    mg_facts = dut.minigraph_facts(host=dut.hostname)['ansible_facts']
+    if not mg_facts['minigraph_portchannels'] or len(mg_facts['minigraph_portchannels']) == 0:
```

Comment:
@noaOrMlnx this check looks redundant: `len(mg_facts['minigraph_interfaces']) == 0`